podcastparser.py

```diff
@@ -632,10 +635,29 @@ def parse_pubdate(text):
     'rss/channel/link': PodcastAttrRelativeLink('link'),
     'rss/channel/description': PodcastAttr('description', squash_whitespace_not_nl),
     'rss/channel/image/url': PodcastAttrRelativeLink('cover_url'),
+    'rss/channel/itunes:author': PodcastAttr('author', squash_whitespace),
     'rss/channel/itunes:image': PodcastAttrFromHref('cover_url'),
     'rss/channel/itunes:type': PodcastAttrType('type', squash_whitespace),
     'rss/channel/atom:link': PodcastAtomLink(),
 
+    'rss/channel/rawvoice:rating': PodcastAttr('rating', squash_whitespace),
+    'rss/channel/rawvoice:liveEmbed': PodcastAttr('liveEmbed', squash_whitespace),
+    'rss/channel/rawvoice:flashLiveStream': PodcastAttr('flashLiveStream', squash_whitespace),
+    'rss/channel/rawvoice:httpLiveStream': PodcastAttr('httpLiveStream', squash_whitespace),
+    'rss/channel/rawvoice:shoutcastLiveStream': PodcastAttr('shoutcastLiveStream', squash_whitespace),
+    'rss/channel/rawvoice:liveStream': PodcastAttr('liveStream', squash_whitespace),
+    'rss/channel/rawvoice:location': PodcastAttr('location', squash_whitespace),
+    'rss/channel/rawvoice:frequency': PodcastAttr('frequency', squash_whitespace),
+    'rss/channel/rawvoice:mycast': PodcastAttr('mycast', squash_whitespace),
+    'rss/channel/rawvoice:poster': PodcastAttr('poster', squash_whitespace),
+    'rss/channel/rawvoice:isHd': PodcastAttr('isHd', squash_whitespace),
+    'rss/channel/rawvoice:embed': PodcastAttr('embed', squash_whitespace),
+    'rss/channel/rawvoice:webm': PodcastAttr('webm', squash_whitespace),
+    'rss/channel/rawvoice:mp4': PodcastAttr('mp4', squash_whitespace),
+    'rss/channel/rawvoice:metamark': PodcastAttr('metamark', squash_whitespace),
+    'rss/channel/rawvoice:subscribe': PodcastAttr('subscribe', squash_whitespace),
+    'rss/channel/rawvoice:donate': PodcastAttr('donate', squash_whitespace),
```

Comment:
The keys in the generated JSON are all `snake_case`, not `camelCase`. Also, feel free to improve the names where it makes sense (e.g. `subscribe` could be `subscription_urls` or whatever, with subkeys for all values).